pkg/config/system_probe.go

```diff
@@ -179,6 +179,7 @@ func InitSystemProbeConfig(cfg Config) {
 
 	cfg.BindEnvAndSetDefault(join(netNS, "enable_gateway_lookup"), true, "DD_SYSTEM_PROBE_NETWORK_ENABLE_GATEWAY_LOOKUP")
 	cfg.BindEnvAndSetDefault(join(netNS, "max_http_stats_buffered"), 100000, "DD_SYSTEM_PROBE_NETWORK_MAX_HTTP_STATS_BUFFERED")
+	cfg.BindEnvAndSetDefault(join(smNS, "max_kafka_stats_buffered"), 100000, "DD_SERVICE_MONITORING_CONFIG_MAX_KAFKA_STATS_BUFFERED")
```

Comment:
You only need to specify the env var name if you aren't using the default (`DD_` + config key)
```suggestion
	cfg.BindEnvAndSetDefault(join(smNS, "max_kafka_stats_buffered"), 100000)
```

Reply:
Why is this under USM config and not DSM?

Reply:
Good point, fixed and added a test

Reply:
It's under USM as the Kafka monitoring feature itself is under the responsibility of the USM monitor (wrongfully called HTTPMonitor at the moment)